src/structures/game_session.ts

```diff
@@ -215,6 +221,7 @@ export default class GameSession {
         this.gameRound = null;
 
         gameRound.interactionMarkAnswers(guessResult.correctGuessers?.length);
+        const guessSpeed = Date.now() - gameRound.startedAt;
```

Comment:
If you move it out here, this means that skips count as well?

Reply:
Should we rename this?

Reply:
Oops, I'll rename it to `timePlayed`